boot/seal.go

```diff
@@ -0,0 +1,85 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
this is missing a matching seal_test.go ?

Reply:
Yes, it's currently being prepared.